src/components/TextField/TextField.tsx

```diff
@@ -31,7 +31,7 @@ interface TextFieldProps extends VibeComponentProps {
   /** See https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete for all of the available options */
   autoComplete?: string;
   value?: string;
-  onChange?: (value: string) => void;
+  onChange?: (value: string, event: React.ChangeEvent | React.MouseEvent) => void;
```

Comment:
on which cases it can be sent as a mouse event?

Reply:
Fixed this one